app/src/main/java/org/oppia/app/topic/reviewcard/ReviewCardListener.kt

```diff
@@ -0,0 +1,7 @@
+package org.oppia.app.topic.reviewcard
+
+/** Allows parent activity to dismiss the [ConceptCardFragment] */
```

Comment:
Update:
`/** Allows parent activity to dismiss the [ReviewCardFragment] */`

Reply:
Done.